automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -153,9 +154,10 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
 
   def randomClusterName: ClusterName = ClusterName(s"automation-test-a${makeRandomId().toLowerCase}z")
 
-  def createNewCluster(googleProject: GoogleProject, clusterName: ClusterName = randomClusterName)(implicit token: AuthToken): Cluster = {
-    val request = ClusterRequest(Map("foo" -> makeRandomId()))
-    val cluster = createAndMonitor(googleProject, clusterName, request)
+  def defaultClusterRequest: ClusterRequest = ClusterRequest(Map("foo" -> makeRandomId()))
```

Comment:
Don't miss this unrelated change.  (Snuck in from my other branch but I'm keeping it)